gui/projectfile.h

```diff
@@ -401,9 +401,11 @@ class ProjectFile : public QObject {
     QStringList getCheckUnknownFunctionReturn() const {
         return mCheckUnknownFunctionReturn;
     }
-    void setCheckUnknownFunctionReturn(const QStringList &s) {
```

Comment:
Why is this a comment?

Reply:
Because the code which uses it is commented out. It's the one I added the TODO to.